crates/exex/src/model.rs

```diff
@@ -87,18 +87,64 @@ impl From<Felt252> for KethMaybeRelocatable {
     }
 }
 
+impl From<u8> for KethMaybeRelocatable {
+    fn from(value: u8) -> Self {
+        Self(Felt252::from(value).into())
+    }
+}
+
 impl From<u64> for KethMaybeRelocatable {
     fn from(value: u64) -> Self {
         Self(Felt252::from(value).into())
     }
 }
 
+impl From<usize> for KethMaybeRelocatable {
+    fn from(value: usize) -> Self {
+        Self(Felt252::from(value).into())
+    }
+}
+
 impl From<Address> for KethMaybeRelocatable {
     fn from(value: Address) -> Self {
         Self::from_bytes_be_slice(&value.0 .0)
     }
 }
 
+/// [`KethMaybeRelocatables`] is a container that holds a vector of [`KethMaybeRelocatable`] values.
+///
+/// This struct is used to represent complex data structures such as Bloom filters or Bytes data in
+/// a format that can be stored and processed by the CairoVM.
+#[derive(Debug, Eq, Ord, Hash, PartialEq, PartialOrd, Clone, Serialize, Deserialize)]
+pub struct KethMaybeRelocatables(Vec<KethMaybeRelocatable>);
+
+impl From<Vec<KethMaybeRelocatable>> for KethMaybeRelocatables {
+    fn from(value: Vec<KethMaybeRelocatable>) -> Self {
+        Self(value)
+    }
+}
+
+impl From<Bloom> for KethMaybeRelocatables {
+    /// Converts a [`Bloom`] filter into a [[`KethMaybeRelocatables`]] structure.
+    ///
+    /// The [`Bloom`] filter is represented as a 256-byte array in big-endian order. However,
+    /// CairoVM's [`Felt252`] type can only handle values up to 252 bits. As a result, we need to
+    /// convert the 256-byte array into smaller pieces that fit within this limit.
+    ///
+    /// Here's how the conversion works:
+    /// - The first element of the `Vec<KethMaybeRelocatable>` is the length of the Bloom filter.
```

Comment:
I don't think we need the length here since it's always 8 Uint256

Reply:
In my new structure that I just pushed, I put 2 fields (len and data) so that we can convert any data and we could use it generically. If we don't want to use it for bloom we will just have to ignore it.